### PR TITLE
Windows compliance + Windows CI + Circle CI

### PR DESCRIPTION
**Pull request by zdne**
Protagonist can now be built & tested on Windows using the combination of `npm install` & `npm test`. 

Furthermore every Protagonist commit / push is **tested**!!!! in AppVeyor CI. Yay!

In addition this PR migrates Protagonist from Travis CI to Circle CI.

UPDATE:

~~**DO NOT MERGE** – there seems to be an issue with Travis-CI and AppVeyor working together. Investigating~~

Travis CI seemed to have problems with AppVeyor of some sort. Moved to CirceCI seems to work. Good to merge.

### CODE REVIEW

**Comment by zdne** [comment]
> Is some specific reason don't use last version 0.10.36?

Copy & paste :) will bump it to 10.36

**Comment by zdne** [comment]
Updated to v0.10.36 and good to merge cc @kuba-kubula  :)

**Comment by kuba-kubula** [comment]
@zdne done and done

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+machine:
+  node:
+    version: 0.10.36
```
Comment (pksunkara) [inline circle.yml]: CircleCI won't let us test on multiple node versions which we do want.

**Pull request merged**